### PR TITLE
Test cases api

### PR DESCRIPTION
**Pull request by abelharisov**

### CODE REVIEW

File: schemas/Project.create.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    enum:
+      - all
+      - group
+      - none
+
+  group:
+    type: string
+    description: Team group hash. Required if access param is set to group.
+
+required: ['title', 'code']
```
Comment (kamilsk) [inline schemas/Project.create.yaml:27]: are you sure you have to use single quote?

Comment (kamilsk) [reply schemas/Project.create.yaml:27]: I checked that `test: [a, b]` is valid YAML and has expected result

**Review by kamilsk — APPROVED** [review]
LGTM, little fixes needed

File: schemas/Project.create.yaml (hunk 1 above)
Comment (abelharisov) [reply schemas/Project.create.yaml]: done

**Pull request merged**